lib/options.nix

```diff
@@ -69,7 +69,12 @@ rec {
           else
             generators.toPretty { multiline = false; } defaultValue;
       in
-      "Plugin default: `${default}`";
+      "Plugin default:"
+      + (
+        # Detect multiline plugin defaults:
+        # FIXME we should trim leading/trailing newlines, to avoid false-positives
```

Comment:
On second thought, this may be a non-issue.

If "false-positives" exist, they're a problem with the calling code, not the implementation here.

Reply:
I agree. I guess you can remove the comment.

Reply:
Done